docs/contributing_tldr.md

```diff
@@ -12,6 +12,9 @@ Please check the [pyproject.toml](https://github.com/commitizen-tools/commitizen
 ### Code Changes
 
 ```bash
+# Make sure you have the latest version of poetry installed
+poetry self update
```

Comment:
This really depends on how poetry is installed. It will not work with system installation, mise or asdf based installation...
So not sure we should include this instruction. 
However, instead of forcing an update which might not be required, saying that "we require `poetry>=X.Y.Z`, you can verify with `poetry --version`" might be a better option